packages/front-end/components/PowerCalculation/stats.ts

```diff
@@ -340,3 +356,294 @@ export function powerMetricWeeks(
   };
   return results;
 }
+
+/*******************/
+export function calculatePriorMean(
+  priorMeanRel: number,
+  mean: number,
+  relative: boolean
+): number {
+  return relative ? priorMeanRel : priorMeanRel * Math.abs(mean);
+}
+
+export function calculatePriorVariance(
+  priorVarianceRel: number,
+  mean: number,
+  relative: boolean
+): number {
+  return relative
+    ? priorVarianceRel
+    : priorVarianceRel * Math.pow(Math.abs(mean), 2);
+}
+
+function calculatePriorMeanSpecified(
+  metric: MetricParams,
+  relative: boolean
+): number {
+  const metricMean = getMetricMean(metric);
+  return calculatePriorMean(metric.priorLiftMean, metricMean, relative);
+}
+
+function calculatePriorVarianceSpecified(
+  metric: MetricParams,
+  relative: boolean
+): number {
+  const metricMean = getMetricMean(metric);
+  return calculatePriorVariance(
+    Math.pow(metric.priorLiftStandardDeviation, 2),
+    metricMean,
+    relative
+  );
+}
+
+function calculatePriorMeanDGP(
+  metric: MetricParams,
+  relative: boolean
+): number {
+  const metricMean = getMetricMean(metric);
+  return calculatePriorMean(metric.effectSize, metricMean, relative);
+}
+
+function calculatePriorVarianceDGP(
```

Comment:
Maybe we can have a comment about why this always returns zero?

Reply:
good call, I added:
  /*priorStandardDeviationDGP is 0 because we assume true fixed effect size*/